sufia-models/app/models/concerns/sufia/works/curation_concern/with_generic_files.rb

```diff
@@ -5,7 +5,7 @@ module WithGenericFiles
 
       included do
         # This used to have a hasFile relation when in hydra-works.  That does not seem to exist so I am using hasPart instead
-        has_many :generic_files, predicate: ActiveFedora::RDF::Fcrepo::RelsExt.isPartOf, class_name: "GenericFile"
+        has_many :generic_files, predicate: ActiveFedora::RDF::Fcrepo::RelsExt.hasPart, class_name: "GenericFile"
```

Comment:
We have not yet integrated HydraWorks into sufia.  This line will be removed once that happens, but for now the stop gap was invalid.